### PR TITLE
Add broadcast intent endpoint

### PR DESCRIPTION
**Pull request by dpgraham**
* Endpoint is `/device/broadcast_intent`
* Takes 'intent' as JSON argument

(related to https://github.com/appium/appium/issues/3172)

### CODE REVIEW

File: lib/mjsonwp/routes.js [hunk 1]
```diff
@@ -369,6 +369,9 @@ const METHOD_MAP = {
   '/wd/hub/session/:sessionId/appium/device/current_package': {
     GET: {command: 'getCurrentPackage'}
   },
+  '/wd/hub/session/:sessionId/appium/device/broadcast_intent': {
```
Comment (mykola-mokhnach) [inline lib/mjsonwp/routes.js]: this is android-specific endpoint. Wouldn't be more appropriate to implement it via executeScript?

Comment (imurchie) [reply lib/mjsonwp/routes.js]: The execute "endpoints" were meant as temporary stop-gaps. There is no reason to make an Android-specific option map there.

**Comment by mykola-mokhnach** [comment]
Can we close this PR now @dpgraham ?

**Comment by dpgraham** [comment]
Yep.

**Pull request closed without merging**